tests/integration/istioio/configuration/scripts/liveness_and_readiness_probes_with_http_annotations.txt

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# $snippet annotate_pod.sh syntax="bash"
+$ kubectl -n istio-io-health apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: liveness-http
+spec:
+  selector:
+    matchLabels:
+      app: liveness-http
+  template:
+    metadata:
+      labels:
+        app: liveness-http
+        version: v1
+      annotations:
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    spec:
+      containers:
+      - name: liveness-http
+        image: docker.io/istio/health:example
+        ports:
+        - containerPort: 8001
+        livenessProbe:
+          httpGet:
+            path: /foo
+            port: 8001
+          initialDelaySeconds: 5
+          periodSeconds: 5
+EOF
+# $endsnippet
+
+# $snippet re-deploy_with_annotation_check_status.sh syntax="bash" outputis="text"
+$ kubectl -n istio-io-health get pod
```

Comment:
You should verify the output here, otherwise you're not really testing whether it works! Same applies to the other snippets where you do `get pod`

Reply:
Cool, I will add that quickly.